crates/egui/src/containers/scroll_area.rs

```diff
@@ -54,6 +54,11 @@ impl State {
     pub fn store(self, ctx: &Context, id: Id) {
         ctx.data_mut(|d| d.insert_persisted(id, self));
     }
+
+    /// Returns true if the scroll area has momentum.
+    pub fn has_momentum(&self) -> bool {
```

Comment:
This can be misinterpreted as to mean "will have momentum if given the chance"

```suggestion
    /// Returns true if the scroll area currently has a velocity due to momentum.
    pub fn has_velocity(&self) -> bool {
```